### PR TITLE
feat(dws): add datasource get availability zones

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

Add new datasource to get the list of DWS availability zones within HuaweiCloud.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new datasource.
2. add related document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccDataSourceAvailabilityZones_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceAvailabilityZones_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceAvailabilityZones_basic
=== PAUSE TestAccDataSourceAvailabilityZones_basic
=== CONT  TestAccDataSourceAvailabilityZones_basic
--- PASS: TestAccDataSourceAvailabilityZones_basic (18.32s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       18.377s```
```
* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_availability_zones_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			},
+		},
+	})
+}
+
+func testDataSourceAvailabilityZones_basic() string {
+	return `
+data "huaweicloud_dws_availability_zones" "test" {}
+`
+}
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_availability_zones_test.go]:
```suggestion
const testDataSourceAvailabilityZones_basic = `data "huaweicloud_dws_availability_zones" "test" {}`
```
The memory cost of the constants are less than variables.

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccDataSourceAvailabilityZones_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceAvailabilityZones_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceAvailabilityZones_basic
=== PAUSE TestAccDataSourceAvailabilityZones_basic
=== CONT  TestAccDataSourceAvailabilityZones_basic
--- PASS: TestAccDataSourceAvailabilityZones_basic (18.41s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       18.497s

File: huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_availability_zones_test.go (hunk 1 above)
Comment (wuzhuanhong) [reply huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_availability_zones_test.go]: done

**Pull request merged**